rewrite-java-core/src/main/java/com/azure/recipes/v2recipes/ContextRecipe.java

```diff
@@ -0,0 +1,61 @@
+package com.azure.recipes.v2recipes;
+
+import org.jetbrains.annotations.NotNull;
+import org.openrewrite.*;
+import org.openrewrite.java.JavaIsoVisitor;
+import org.openrewrite.java.tree.J;
+import org.openrewrite.java.tree.TypeTree;
+
+/**
+ * ContextRecipe changes all instances of Context.NONE (from azure core v1) to Context.none() (from azure core v2).
+ * This recipe also updates the import statements for the aforementioned class.
+ * @author Ali Soltanian Fard Jahromi
+ */
+public class ContextRecipe extends Recipe {
```

Comment:
Let's use the `ChangeType` and `ChangeStaticFieldToMethod` recipes for this. The latest version of OpenRewrite ignores the module-info.java class as a temporary solution. So, there will eventually be a fix for this. 

Also, we'll need to use `ChangeType` and `ChangeStaticFieldToMethod` in a few other places. So, we don't want to add custom recipes for all those cases. 

For "FindAndReplace" recipe to update module-info, it's a much simpler string replacement and can be done as custom recipe for the short-term (until OpenRewrite fixes the root cause).